Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.9-slim
+
+# Set the working directory in the container
+WORKDIR /Brazillian_Orders_Peter
+
+#install dependencies
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+# --no-cache-dir 
+#copy the entrypoint.sh file to the container
+COPY entrypoint.sh /Brazillian_Orders_Peter/entrypoint.sh
+RUN chmod +x /Brazillian_Orders_Peter/entrypoint.sh
+
+#copy the data directory to the container
+COPY data /Brazillian_Orders_Peter/data
+
+EXPOSE 8080
+#copy the entire src directory, which includes the main files
+COPY src /Brazillian_Orders_Peter/src
+#copy the entire tests directory, which includes the test files
```

Comment:
too many comments in this file, kindly comment on only things that are difficult to understand or remember OR things that are not obvious to grasp by people who did not write it with you.

Reply:
Alright, Noted